joint_limits:
  body_roll_joint:
    has_position_limits: true
    min_position: -1.57
    max_position: 1.57
    has_velocity_limits: true
    max_velocity: 1.0
    has_acceleration_limits: true
    max_acceleration: 1.0
  head_tilt_joint:
    has_position_limits: true
    min_position: -1.57
    max_position: 1.57
    has_velocity_limits: true
    max_velocity: 1.0
    has_acceleration_limits: true
    max_acceleration: 1.0
  head_pan_joint:
    has_position_limits: true
    min_position: -2.62
    max_position: 2.62
    has_velocity_limits: true
    max_velocity: 1.0
    has_acceleration_limits: true
    max_acceleration: 1.0
  right_shoulder_roll_joint:
    has_position_limits: true
    min_position: -1.57
    max_position: 1.57
    has_velocity_limits: true
    max_velocity: 1.0
    has_acceleration_limits: true
    max_acceleration: 1.0
  right_shoulder_flex_joint:
    has_position_limits: true
    min_position: -1.57
    max_position: 1.57
    has_velocity_limits: true
    max_velocity: 1.0
    has_acceleration_limits: true
    max_acceleration: 1.0
  right_elbow_roll_joint:
    has_position_limits: true
    min_position: -1.57
    max_position: 1.57
    has_velocity_limits: true
    max_velocity: 1.0
    has_acceleration_limits: true
    max_acceleration: 1.0
  right_hand_motor_joint:
    has_position_limits: true
    min_position: 0
    max_position: 1.57
    has_velocity_limits: true
    max_velocity: 1.0
    has_acceleration_limits: true
    max_acceleration: 1.0
  left_shoulder_roll_joint:
    has_position_limits: true
    min_position: -1.57
    max_position: 1.57
    has_velocity_limits: true
    max_velocity: 1.0
    has_acceleration_limits: true
    max_acceleration: 1.0
  left_shoulder_flex_joint:
    has_position_limits: true
    min_position: -1.57
    max_position: 1.57
    has_velocity_limits: true
    max_velocity: 1.0
    has_acceleration_limits: true
    max_acceleration: 1.0
  left_elbow_roll_joint:
    has_position_limits: true
    min_position: -1.57
    max_position: 1.57
    has_velocity_limits: true
    max_velocity: 1.0
    has_acceleration_limits: true
    max_acceleration: 1.0
  left_hand_motor_joint:
    has_position_limits: true
    min_position: 0
    max_position: 1.57
    has_velocity_limits: true
    max_velocity: 1.0
    has_acceleration_limits: true
    max_acceleration: 1.0